pkg/client/events_query/client_test.go

```diff
@@ -0,0 +1,413 @@
+package eventsquery_test
```

Comment:
:disappointed: Mistakes were made; this file wasn't meant to be committed until 66fdc790eca790c4173afcb8e7d6adffeff0c6e0. There will be modifications in that commit.